### DISCUSSION TITLE
No me pasan mi nueva contraseña

### DISCUSSION THREAD

**Discussion by FelipeBENIITEZ** [Launch-X-Latam/MisionFrontEnd#463] [Team Posts]
hace dos semanas que pedi que me reestablezcan la contraseña de mi cuenta innovaccion, ya que de un momento a otro me cerro sesion y no me aceptaba la contraseña que me proporcionaron en un inicio, pedi que me la reestablezcan me dijeron que lo hicieron pero nunca me pasaron las nueva contraseña llevo dias mandando correos y enviando mensajes en el instagram de innovaccion.
Ya no se que hacer

**Comment by SujeiRodriguez** [comment]
a mi me paso que me estaba metiendo en la pag de microsoft que no era, debe ser microsoft 365 y me pasaron el enlace por instagram, no te pasaron ningun enlace?

**Comment by FelipeBENIITEZ** [comment]
yo si entre donde debia la cuestion es que me reestablecieron la contraseña y no me la han proporcionado
